cli/cloe_launch/binutils.py

```diff
@@ -0,0 +1,59 @@
+import logging
```

Comment:
should a copyright header and license notice be added here?

Reply:
Fixed.